source/_integrations/broadlink.markdown

```diff
@@ -34,10 +37,21 @@ The entities have the same name as the device by default. To change the name, ic
 
 The entities are divided into three subdomains:
 
+- [Climate](#climate)
 - [Remote](#remote)
 - [Sensor](#sensor)
 - [Switch](#switch)
 
+## Climate
+
+The `climate` entities allow you to monitor and control Broadlink thermostats.
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: broadlink
+    mac: MAC_ADDRESS
```

Comment:
We no longer use YAML to set up platforms. They are created automatically when the user adds the device.

Reply:
Can you please help me out on that?
When I try to add the climate device via the frontend I get the following log message:
```python
2021-04-14 19:24:28 ERROR (MainThread) [homeassistant.components.climate] Error adding entities for domain climate with platform broadlink
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity_platform.py", line 322, in async_add_entities
    await asyncio.gather(*tasks)
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity_platform.py", line 514, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity.py", line 546, in add_to_platform_finish
    await self.async_added_to_hass()
  File "/home/homeassistant/.homeassistant/custom_components/broadlink/climate.py", line 113, in async_added_to_hass
    if state.state:
AttributeError: 'NoneType' object has no attribute 'state'
2021-04-14 19:24:28 ERROR (MainThread) [homeassistant.components.climate] Error while setting up broadlink platform for climate
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity_platform.py", line 213, in _async_setup_platform
    await asyncio.gather(*pending)
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity_platform.py", line 322, in async_add_entities
    await asyncio.gather(*tasks)
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity_platform.py", line 514, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity.py", line 546, in add_to_platform_finish
    await self.async_added_to_hass()
  File "/home/homeassistant/.homeassistant/custom_components/broadlink/climate.py", line 113, in async_added_to_hass
    if state.state:
AttributeError: 'NoneType' object has no attribute 'state'
```

So it seems like it can't find the device at all. I double checked the IP.

Reply:
That seems unrelated to the documentation PR @majuss